2018-01-10 11:09:44,213 INFO [br.com.bradesco.web.aq.application.log.impl.LogManagerImpl] - [Loaded LOG CATALOG from EXTERNAL Location [ '/suportedbdc_apl/custodia/calisessaoWS/config/log4j-catalog.xml'].]
2018-01-10 11:09:45,967 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -e84b7f8:160d59c0189:-7fc7:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 11:12:06,675 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -e84b7f8:160d59c0189:-7fc6:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 11:12:07,145 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -e84b7f8:160d59c0189:-7fc3:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 11:12:07,403 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -e84b7f8:160d59c0189:-7fc4:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 16:28:10,556 INFO [br.com.bradesco.web.aq.application.log.impl.LogManagerImpl] - [Loaded LOG CATALOG from EXTERNAL Location [ '/suportedbdc_apl/custodia/calisessaoWS/config/log4j-catalog.xml'].]
2018-01-10 16:28:10,594 WARN [br.com.bradesco.web.aq.application.alert.config.AlertCatalogImpl] - [Could not open ServletContext resource [/WEB-INF/classes/alert-catalog.xml]]
2018-01-10 16:28:10,601 WARN [br.com.bradesco.web.aq.application.alert.channel.file.impl.FileEmitterImpl] - [No such file or directory]
2018-01-10 16:33:56,415 INFO [br.com.bradesco.web.aq.application.log.impl.LogManagerImpl] - [Loaded LOG CATALOG from EXTERNAL Location [ '/suportedbdc_apl/custodia/calisessaoWS/config/log4j-catalog.xml'].]
2018-01-10 16:33:59,705 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -65d4d392:160e153de73:-7ffb:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 16:33:59,915 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -65d4d392:160e153de73:-7ff9:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
2018-01-10 16:34:00,106 WARN [br.com.bradesco.web.aq.application.object.context.DataContextHolderImpl] - [Obtido dataContext já em uso com ID: -65d4d392:160e153de73:-7ff8:172.16.43.93 no método createDataContext() do ObjectManager. Não reutilizamos, criamos novo dataContext para a request recebida: reqSessionId[null], requestUrl[http://www.bc2.custodia.des.scopus.com.br/calisessaows/SessaoWSService]]
